### DISCUSSION TITLE
Cloning, Multiple Accounts

### DISCUSSION THREAD

**Discussion by rogersm-uwosh** [community/community#38509] [New to GitHub]
I have a student who is trying to follow these instructions -- https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/managing-multiple-accounts?platform=windows

But when we try and clone a repo, it doesn't ask for credentials, although the instructions says it should.

Any idea why?

**Comment by david-engelmann** [comment]
Do you have ssh authentication setup with the github account?

**Reply by ghost** [reply]
Yes

**Comment by spenserblack** [comment]
What's the output of `git config credential.helper`? If the student has used their credentials once before, it's likely that the credential helper has stored the credentials for later use.

**Reply by ghost** [reply]
it doesn't output anything

**Comment by samus-aran** [comment]
Hi @choiw56 & @rogersm-uwosh 👋 
How did you and your student get on?

**Reply by mprogers** [reply]
They were unable to get this to work, so they ended up relying on a partner. Not ideal, but it happens.
